packages/phone-number-privacy/signer/src/tracing.ts

```diff
@@ -0,0 +1,64 @@
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger'
+import { registerInstrumentations } from '@opentelemetry/instrumentation'
+import { Resource } from '@opentelemetry/resources'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node')
+//import { WebTracerProvider } from '@opentelemetry/sdk-trace-web'
```

Comment:
I added some expanded comments to explain those lines. 
Not needed but just a ref. on how to add specific instrumentation for auto-tracing instead of the all-included `getNodeAutoInstrumentations`